src/components/pages/Form.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import emailjs from 'emailjs-com';
+import styled from 'styled-components';
+import { Content, Container } from './PagesStyles';
+
+const FormContent = styled.div`
+	@media screen and (min-width: 780px) {
+		display: flex;
+		margin-left: 5rem;
+	}
+`;
+const Form = styled.form`
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	width: 100%;
+
+	> input {
+		height: 35px;
+		border-radius: 4px;
+		border: 1px solid #efece1;
+	}
+
+	.submit-button {
+		background-color: #efece1;
+	}
+
+	> textarea {
+		height: 150px;
+		border-radius: 4px;
+		border: 1px solid #efece1;
+		margin-bottom: 0.5rem;
+	}
+
+	@media screen and (min-width: 780px) {
+		max-width: 400px;
+		margin: 0 2rem;
+	}
+`;
+
+const Info = styled.div`
+	font-size: 14px;
+	padding: 10px;
+	color: #747679;
+
+	.address {
+		padding: 10px 0;
+
+		> p {
+			padding: 4px 0;
+		}
+	}
+`;
+
+const ContactUs = () => {
+	const sendEmail = (e) => {
+		e.preventDefault();
+		emailjs.sendForm('default_service', 'contatc_form', e.target, 'user_9M071Z9YysFIbLj7JLEaI').then(
```

Comment:
i think its good to have a ui change like, if the email is succesful?

Reply:
yes you're right. I just added ;)